tools/SourceKit/tools/sourcekitd-test/TestOptions.cpp

```diff
@@ -290,5 +294,11 @@ bool TestOptions::parseArgs(llvm::ArrayRef<const char *> Args) {
     return true;
   }
 
+  if (Timings) {
+    PrintRequest = false;
+    PrintResponseAsJSON = false;
+    PrintRawResponse = false;
+  }
```

Comment:
This seems like a reasonable default for `-timings`, but if someone specifies `-timings -print-request ...` then I think we should print the request :-)